_config/userforms.yml

```diff
@@ -7,3 +7,6 @@ LeftAndMain:
 
   extra_requirements_css:
     - userforms/css/FieldEditor.css
+Director:
+  rules:
+    'UserDefinedForm_Controller//$Action': 'UserDefinedForm_Controller'
```

Comment:
I don't like the idea of exposing the entire controlled just for the ping. Can we lock it down more specifically?